dev/Projection/Microsoft.UI.Xaml.Projection.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+   <PropertyGroup>
+      <TargetFramework>net8.0-windows10.0.22621.0</TargetFramework>
+      <Platform>AnyCPU</Platform>
+      <UseUwp>true</UseUwp>
+      <DisableRuntimeMarshalling>true</DisableRuntimeMarshalling>
```

Comment:
Wait, you _also_ need `IsAotCompatible` 😅
I just meant delete the first two, not that one too, my bad.